lib/db/util.js

```diff
@@ -7,6 +7,7 @@
 const crypto = require('crypto')
 const P = require('../promise')
 const randomBytes = P.promisify(require('crypto').randomBytes)
+const scryptHash = P.promisify(require('scrypt-hash'))
```

Comment:
I don't see any local tests for this module, should we add some for the stuff you're adding here? Seems like something that could be valuable for us to add to going forward, given the fundamental nature of some of this code?

Reply:
(but no worries if you think testing via the db methods is sufficient coverage)